data-types.md

```diff
@@ -199,9 +195,13 @@ A Trip is defined by the following structure:
 | Field                    | Type            | Required/Optional      | Comments |
 | -----                    | ----            | -----------------      | -------- |
 | `provider_id`            | UUID            | Required               | A UUID for the Provider, unique within MDS. See MDS [provider list](/providers.csv). |
+| `provider_name` | String | Required | The public-facing name of the Provider |
 | `data_provider_id`       | UUID            | Optional               | If different than `provider_id`, a UUID for the data solution provider managing this data endpoint. See MDS [provider list](/providers.csv) which includes both service operators and data solution providers. |
 | `device_id`              | UUID            | Required               | A unique device ID in UUID format |
-| `journey_id`             | UUID            | Optional               | A unique [journey ID](/modes#journey-id) for associating collections of trips for its [mode] |
+| `vehicle_id` | String | Required | The Vehicle Identification Number visible on the vehicle itself |
```

Comment:
Same argument here. We have a device registry, and the whole point was to NOT include redundant info over and over and over.

Reply:
I guess since other data in /vehicles now like vehicle_attributes, propulsion_types, and accessibility_options, then fields like vehicle_id and vehicle_type could also be removed from /trips.